recipes/pyre-extensions/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyre-extensions" %}
+{% set version = "0.0.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyre-extensions-{{ version }}.tar.gz
+  sha256: df019263900af45e83e41fb9e889824e8b7bcdb9cb67fe2957bb80104eadcbc7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
Can this one be noarch?

Reply:
`pyre-extensions` could be `noarch` but on windows it is raising the following error. So for now I build it only for `linux` and `mac os x`. For more details check #22066

```
Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "C:\bld\pyre-extensions_1676686962353\work\setup.py", line 5, in <module>
      long_description = fh.read()
                         ^^^^^^^^^
    File "C:\bld\pyre-extensions_1676686962353\_h_env\Lib\encodings\cp1252.py", line 23, in decode
      return codecs.charmap_decode(input,self.errors,decoding_table)[0]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 76: character maps to <undefined>
  error: subprocess-exited-with-error
```

Reply:
I think that is fine for noarch. It will be installable, that only affects the build but the noarch pkgs are built on Linux anyway.

Reply:
Ok, I switched the `pyre-extensions` package to `noarch`